specs/monitoring_integration.adoc

```diff
@@ -0,0 +1,390 @@
+= Spec - Introducing a new component monitoring-integration
+
+A new component called Monitoring-Integration is introduced which integrates Grafana
+with tendrl core.
+
+
+== Problem description
+
+The new monitoring-integration component would primarily focus on dynamic
+configuration of grafana, based on the topology changes in etcd. Some of these
+responsibilities would be:
+
+* monitoring-integration should store predefined json’s for dashboards, panels and alerts.
```

Comment:
Monitoring Integration stores json for default dashboard which contains details about panels, alerts and metrics